src/common/components/CoreLayout/Sidebar/SidebarItem.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+import { type SidebarListItem } from "./types";
+
+export const SidebarItem = ({
+  label,
+  icon,
+  href,
+  external = false,
+  active,
+}: SidebarListItem) => {
+  const SidebarLink = external ? "a" : Link;
+  return (
+    <li>
+      <SidebarLink
+        href={href}
+        className={
+          "flex items-center gap-x-3 rounded-md px-3 py-2 hover:bg-gradient-to-r hover:from-element-secondary" +
+          (active ? " bg-gradient-to-r from-element-secondary" : "")
+        }
```

Comment:
Lets also add some simple transition animations to make the bg transition more pleasant

Maybe you can try this `transition-colors duration-200 ease-out`

Reply:
TIL CSS does not allow us to add transitions to gradients directly. My workaround is to add an outer div with the gradient and add transition to the inner div's bg

Reply:
HAHA that's right. Forgot about that